test/message_test.rb

```diff
@@ -50,4 +55,18 @@ def test_that_it_has_a_velocity
     message = ADSB::Message.new('8D485020994409940838175B284F')
     assert_in_delta(159.201, message.velocity)
   end
+
+  def test_that_it_is_odd
+    message = ADSB::Message.new(positionmessage)
+    assert_equal(false, message.odd?)
+  end
+
+  def test_that_it_is_even
+    message = ADSB::Message.new(positionmessage)
+    assert_equal(true, message.even?)
+  end
+  def test_that_the_cprflag_is
```

Comment:
I think there is something missing at the end. Should it be `test_that_the_cpr_flag_is_even`?